mobile/src/screens/components/Board.tsx

```diff
@@ -0,0 +1,84 @@
+// Initial chessboard code credits go to William Candillon
+// Github: https://github.com/wcandillon
+// Source Code: https://github.com/wcandillon/can-it-be-done-in-react-native/tree/master/season4/src/Chess
+import { Chess } from "chess.js";
+import React, { useCallback, useRef, useState } from "react";
+import { View, StyleSheet, Dimensions, Text } from "react-native";
+
+import Background from "./Background.tsx";
+import Gameover from "./Gameover.tsx";
+import Piece from "./Piece.tsx";
+
+const { width } = Dimensions.get("window");
+
+function useConst<T>(initialValue: T | (() => T)): T {
+  const ref = useRef<{ value: T }>();
+  if (ref.current === undefined) {
+    // Box the value in an object so we can tell if it's initialized even if the initializer
+    // returns/is undefined
+    ref.current = {
+      value:
+        typeof initialValue === "function"
+          ? // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Can you explain why you're disabling this rule here?

Reply:
It was in the source code, disabling it returns 

Don't use `Function` as a type. The `Function` type accepts any function-like value.
It provides no type safety when calling the function, which can be a common source of bugs.
It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
If you are expecting the function to accept certain arguments, you should explicitly define the function shape  @typescript-eslint/ban-types

Reply:
Let's leave this since it was in the source code. We should be safe from this since we aren't going to use `useConst` ourselves.